src/components/Quotes/Quotes.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+import Slider from "react-slick";
+
+const settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
```

Comment:
if fade is not set to true `fade: true` what is the use of `speed`?

Reply:
```speed``` is the animation speed